pkg/handlers/resourceMappings.go

```diff
@@ -39,13 +39,18 @@ func (h *Handler) GetResourceMapping(id string) (*policy.ResourceMapping, error)
 	return res.GetResourceMapping(), nil
 }
 
-func (h *Handler) ListResourceMappings() ([]*policy.ResourceMapping, error) {
-	res, err := h.sdk.ResourceMapping.ListResourceMappings(context.Background(), &resourcemapping.ListResourceMappingsRequest{})
+func (h *Handler) ListResourceMappings(limit, offset int32) ([]*policy.ResourceMapping, *policy.PageResponse, error) {
+	res, err := h.sdk.ResourceMapping.ListResourceMappings(context.Background(), &resourcemapping.ListResourceMappingsRequest{
```

Comment:
And why does this one use `context.Background()`?  Shouldn't they all use `h.ctx` to be consistent?